drivers/telescope/telescope_simulator.cpp

```diff
@@ -447,13 +446,13 @@ bool ScopeSim::ISNewSwitch(const char *dev, const char *name, ISState *states, c
     if (dev != nullptr && strcmp(dev, getDeviceName()) == 0)
     {
 #ifdef USE_SIM_TAB
-        if (strcmp(name, mountTypeSP.name) == 0)
+        if (strcmp(name, mountTypeSP.getName()) == 0)
         {
-            if (IUUpdateSwitch(&mountTypeSP, states, names, n) < 0)
+            if (IUUpdateSwitch(mountTypeSP, states, names, n) < 0)
                 return false;
 
-            mountTypeSP.s = IPS_OK;
```

Comment:
there was a confusion of mountTypeSP.s
It should be used as a property?

Reply:
mountTypeSP.setState(IPS_OK)